vllm/worker/hpu_model_runner.py

```diff
@@ -894,12 +908,11 @@ def _maybe_compile(self, *args, **kwargs):
                                   'false').strip().lower() in ("1", "true")
             if os.getenv('VLLM_REGIONAL_COMPILATION',
                          'true').strip().lower() in ("1", "true"):
-                compiled_methods = [self.model._set_block_mapping]
-                for method in compiled_methods:
-                    method = torch.compile(method,
-                                           backend='hpu_backend',
-                                           fullgraph=fullgraph,
-                                           dynamic=False)
+                self.model._set_block_mapping = torch.compile(
```

Comment:
please use setattr(model, name, function) as we discuss offline.
Where names will be store in in the list (instead of compiled_methods list)